templates/zerver/help/include/mobile-profile-menu.md

```diff
@@ -0,0 +1 @@
+1. From the home screen, tap your **profile picture** in the lower right.
```

Comment:
Hm, I don't think the "home screen" is a well-defined concept... We have the bottom nav on a number of screens, some of which I definitely would not call a home screen (e.g. streams)

It might be best to just say, "Tap your **profile picture** in the lower right," or maybe "Tap your **profile picture** in the bottom right corner."

@gnprice what do you think?

Reply:
Good point. Yeah, "Tap your profile picture in the bottom right corner of the screen." would be best, I think.

Reply:
I know we had "corner of the screen" phrasing elsewhere, but I wonder if we should standardize on "corner of the app"? E.g. the user might be doing a split screen...

Reply:
Yes, I think it'd be a good idea to standardize on "corner of the app" because it's more specific and we can use it to refer to the web app, desktop app, or mobile app.

Reply:
Cool, could you check for and fix any uses of `screen` if you haven't done so already?